### DISCUSSION TITLE
Is transparency broken on windows 11 (dev build)?

### DISCUSSION THREAD

**Discussion by iHeadway** [Flow-Launcher/Flow.Launcher#2112] [Q&A]
I modded the "Circle System" theme to support transparency, but the corners don't render properly (I have the same results with other transparent themes)
![Flow Launcher_mJlrBJ2CRq](https://user-images.githubusercontent.com/30590849/235984632-0159c7c8-d538-4aa5-99b5-7b3732c93b0c.png)
and often the results window does something like "repeat-y" from the search line for the background
![Flow Launcher_qRqdkEGrZu](https://user-images.githubusercontent.com/30590849/235985671-5a0e4122-79b1-4429-9d39-94410beee580.png)

**Comment by onesounds** [comment]
Sorry for the delayed response. There is a slight issue with the translucency feature. The translucency feature extends to the outside of the window, causing shadow and rounded corner issues. To use it, you need to use rectangles, not rounded corners. We've tried to improve this several times, but haven't succeeded yet. 😂
